pages/linux/wol.md

```diff
@@ -0,0 +1,28 @@
+# wol
+
+> Client for sending Wake-on-LAN magic packets.
+> More information: <https://sourceforge.net/projects/wake-on-lan/>.
+
+- Send a WoL packet to a device:
+
+`wol {{mac_address}}`
+
+- Send a WoL packet to a device in another subnet based on its IP:
+
+`wol --ipaddr={{ip_address}} {{mac_address}}`
+
+- Send a WoL packet to a device in another subnet based on its hostname:
+
+`wol --host={{hostname}} {{mac_address}}`
+
+- Send a WoL packet to a specific port:
+
+`wol --port={{port_number}} {{mac_address}}`
+
+- Read hardware addresses, IP addresses/hostnames, optional ports and SecureON password from a file:
```

Comment:
Is 'password' supposed to plural in this case? It if it, then use "passwords". If singular is intended, then prefix "SecureON" with "the" or "a", e.g.:
```suggestion
- Read hardware addresses, IP addresses/hostnames, optional ports and the SecureON password from a file:
```

Reply:
I'd assume each device can have its own password. I copied the description straight from `man` since it was such a neat description.

Reply:
Okay, that's great as long as it's clear.  Before merging this I'd like to definitely understand this.